addons/content-browse/ui/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "description": "react practice project",
   "main": "src/main/js/index.js",
-  "license": "MIT",
+  "license": "Apache-2.0",
```

Comment:
"description": "react practice project" above... :grinning:

Reply:
oops.... forgot to change.